config/templates/config-docker.conf

```diff
@@ -106,10 +106,10 @@ If you prefer to use profile, for example, `userpatches/config-my.conf`,  try:
     ./compile.sh my compile_uboot
 
 EOF
-  docker run "${DOCKER_FLAGS[@]}" -it --rm --entrypoint /usr/bin/env armbian:$VERSION "$@" /bin/bash
+  docker run "${DOCKER_FLAGS[@]}" -it --entrypoint /usr/bin/env armbian:$VERSION "$@" /bin/bash
 else
   display_alert "Running the container" "" "info"
-  docker run "${DOCKER_FLAGS[@]}" -it --rm armbian:$VERSION "$@"
```

Comment:
Is that -rm placement justified in first place?

Reply:
Formayl yes, but it doesn't matter.
`--rm` just moves from hardcoded command line in line 112 into `DOCKER_FLAGS[@]` array as option from line 46